Assets/SEETests/TestGraphProviderIO.cs

```diff
@@ -254,6 +259,75 @@ private static void AreEqualDiffMergeGraphProviders(MergeDiffGraphProvider saved
 
         #endregion
 
+        private static T NewVanillaSEECity<T>() where T : Component
+        {
+            return new GameObject().AddComponent<T>();
+        }
+
+        public async System.Threading.Tasks.Task<Graph> GetVCSGraphAsync()
+        {
+            VCSGraphProvider saved = GetVCSGraphProvider();
+            SEECity testCity = NewVanillaSEECity<SEECity>(); ;
+            Graph testGraph = new("test", "test");
+            Graph graph = await saved.ProvideAsync(testGraph, testCity);
+            return graph;
+        }
+
+        [Test]
+        public void TestVCSGraphProvider()
+        {
+            Graph graph = GetVCSGraphAsync().GetAwaiter().GetResult();
```

Comment:
Have you noticed the following warning regarding this code:
`Synchronously waiting on tasks or awaiters may cause deadlocks. Use await or JoinableTaskFactory.Run instead.`

You are trying to run asynchronous code synchronously. Instead you need to turn your test into an `async ` test and return `Task` instead of `void`.